__tests__/server/organization.js

```diff
@@ -0,0 +1,49 @@
+/* globals afterAll, describe, expect */
+
+import mongoose from 'mongoose'
+
+import {Organization} from '../../server/models'
+
+import {makeRestEndpointTests} from '../test-utils/server'
+
+describe('organization', () => {
+  afterAll(() => {
+    mongoose.disconnect() // disconnect from mongo to end running of tests
+  })
+
+  const initOrganizationData = {
+    agencyId: mongoose.Types.ObjectId(),
+    name: 'test-org'
+  }
+
+  makeRestEndpointTests({
+    endpoints: {
+      'Collection GET': {},
+      'Collection POST': {
+        creationData: initOrganizationData,
+        customAssertions: (json) => {
```

Comment:
The configuration and custom assertions here are small compared to the boilerplate code that doesn't need to be written as a result.  And they force consistency across the app.